Documentation/deploy-master.md

```diff
@@ -1,6 +1,6 @@
 # Deploy Kubernetes Master Node(s)
 
-Boot a single CoreOS machine which will be used as the Kubernetes master node. You must use a CoreOS version 773.1.0+ on the Alpha or Beta channel for the `kubelet` to be present in the image.
+Boot a single CoreOS machine which will be used as the Kubernetes master node. You must use a CoreOS version 960.0.0+ for the `kubelet-wrapper` script to be present in the image. If you wish to use an earlier version (e.g. from the 'stable' channel) see [kubelet-wrapper](kubelet-wrapper.md) for more information.
```

Comment:
My gut says that this mentions to many "under the hood" details, ie mentioning the wrapper at all. Maybe just say:

"You must use a CoreOS version 960.0.0+ for the correct version of the Kubelet to be in the image."